src/layouts/Login/LoginPage.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Divider,
+  Flex,
+  GridItem,
+  GridProps,
+  Text,
+  Grid,
+  HStack,
+  VStack,
+  Tabs,
+  TabList,
+  TabPanels,
+  TabPanel,
+} from "@chakra-ui/react"
+import {
+  Button,
+  GovtMasthead,
+  Tab,
+  InlineMessage,
+  Link,
+} from "@opengovsg/design-system-react"
+import { useEffect, useState, PropsWithChildren } from "react"
+
+import { useLogin, useVerifyOtp } from "hooks/loginHooks"
+
+import { useSuccessToast } from "utils/toasts"
+
+import { IsomerLogo, LoginImage, OGPLogo } from "assets"
+
+import { LoginForm, LoginProps, OtpForm, OtpProps } from "./components"
+
+const LOGIN_GRID_LAYOUT: Pick<
+  GridProps,
+  | "gridTemplateAreas"
+  | "gridTemplateColumns"
+  | "gridTemplateRows"
+  | "height"
+  | "width"
+> = {
+  gridTemplateAreas: `"image content" 
+                      "credits links"`,
+  gridTemplateColumns: "2fr 4fr",
+  gridTemplateRows: "1fr 5rem",
+  height: "100%",
+  width: "100%",
+}
+
+interface FooterLinkProps {
+  link: string
+}
+
+const FooterLink = ({
+  link,
+  children,
+}: PropsWithChildren<FooterLinkProps>): JSX.Element => (
+  <Link
+    href={link}
+    rel="noopener noreferrer"
+    target="_blank"
+    textDecorationLine="none"
+    color="text.link.dark"
+    _hover={{
+      color: "text.link.dark",
+      textDecorationLine: "underline",
+    }}
+  >
+    {children}
+  </Link>
+)
+
+const LoginContent = (): JSX.Element => {
+  const { mutateAsync: sendLoginOtp, error: loginError } = useLogin()
+
+  const { mutateAsync: verifyLoginOtp, error: verifyError } = useVerifyOtp()
+
+  const successToast = useSuccessToast()
+  const [email, setEmail] = useState<string>("")
+  const [loginErrorMessage, setLoginErrorMessage] = useState("")
+  const [verifyErrorMessage, setVerifyErrorMessage] = useState("")
+
+  const handleSendOtp = async ({ email: emailInput }: LoginProps) => {
+    const trimmedEmail = emailInput.trim()
+    await sendLoginOtp({ email: trimmedEmail })
+    successToast({
+      description: `OTP sent to ${trimmedEmail}`,
+    })
+    setEmail(trimmedEmail)
+  }
+
+  const handleVerifyOtp = async ({ otp }: OtpProps) => {
+    await verifyLoginOtp({ email, otp })
+    window.location.reload()
```

Comment:
Easiest way to redirect to sites!

Reply:
if we should **always** redirect to `/sites` on a successful login, we might wanna hide this inside the `useVerifyOtp` hook so that consumers don't have to go through this drudgery. 

also, we could use the `useHistory` hook from `react-router` and do `history.push("/sites")` to save us this detail as well

Reply:
Fixed in c2dd5da4 - thanks for debugging with me offline!